doc/index.rst

```diff
@@ -64,6 +64,13 @@ Jump right in:
    Running CLIMADA on Euler <guide/Guide_Euler>
 
 
+.. toctree::
+   :caption: API Reference
+   :hidden:
+   
+   Python Modules <climada/climada>
```

Comment:
Why not rename "Python Modules" to "API Reference", and leave it in the original toctree? I would not add this as a separate toctree with caption because I don't see that we add any other entries in the future. And a separate caption for only a single item does not make much sense to me

Reply:
1) It definitely deserves its own section. (I agree that a single item is a wee bit awkward. I wanted to make the section title  [API Reference] interactive and unfold subsections only when I actually click on it, but I couldn't figure out how to do that.)
2) For making navigation easier. When I browse through the API reference and its navigation is at the bottom of the page I lose context through re-rendering whenever I click on a module. When it's on top, the unfolded doc-tree is stable.